UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartTraumaDamage.cs

```diff
@@ -272,15 +272,16 @@ private BodyPartCutSize GetCutSize(TraumaDamageLevel traumaLevel)
 		private TraumaDamageLevel CheckTraumaDamageLevels(float traumaDamage)
 		{
 			//(Max) : Later we should add scaling values based on the body part's MaxHP.
-			switch (traumaDamage)
+
+			switch ((int)traumaDamage)
 			{
-				case float n when n.IsBetween(0, 25, false):
+				case int n when n.IsBetween(0, 25, true):
```

Comment:
if it is inclusive, no need for true. That's the default value. Also they all can  be inclusive since we're casting to int

Reply:
the true is there to make it more understandable, also since this is all ints we can just remove the inclusive bool from all?

Reply:
ye

Reply:
![image](https://user-images.githubusercontent.com/34368774/133718218-e4215578-b7ef-445d-9e3c-5668fedb18cf.png)
I forgot to press this 2 days ago..

Reply:
review pls